client/src/sockets/enums.js

```diff
@@ -41,19 +41,19 @@ export const ListEvents = Object.freeze({
   REMOVE_BY_IDS: ListActionTypes.REMOVE_BY_IDS,
   REMOVE_MEMBER_ROLE_SUCCESS: ListActionTypes.REMOVE_MEMBER_ROLE_SUCCESS,
   REMOVE_MEMBER_SUCCESS: ListActionTypes.REMOVE_MEMBER_SUCCESS,
-  REMOVE_ON_ARCHIVE_COHORT: 'list/REMOVE_ON_ARCHIVE_COHORT',
+  REMOVE_WHEN_COHORT_UNAVAILABLE: 'list/REMOVE_WHEN_COHORT_UNAVAILABLE',
```

Comment:
what those ''REMOVE_WHEN_COHORT_UNAVAILABLE' actions mean??

Reply:
remove the cohort from store when cohort is unavaiable that means when is archved or deleted.(REMOVE_ON_COHORT_DELETION_ARCHIVIZATION?)
DELETE_SUCCES is emitting in many other situations without redirecting so I had to use another action I tried to use ARCHIVED_SUCCESS in deletion case but I look strange. But I agree that this name can be misleading.